install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -64,31 +67,35 @@ spec:
     rules:
     - alert: ClusterNotUpgradeable
       annotations:
-        message: One or more cluster operators have been blocking minor version cluster upgrades for at least an hour for reason {{ "{{ with $cluster_operator_conditions := \"cluster_operator_conditions\" | query}}{{range $value := .}}{{if and (eq (label \"name\" $value) \"version\") (eq (label \"condition\" $value) \"Upgradeable\") (eq (label \"endpoint\" $value) \"metrics\") (eq (value $value) 0.0) (ne (len (label \"reason\" $value)) 0) }}{{label \"reason\" $value}}.{{end}}{{end}}{{end}}"}} {{ "{{ with $console_url := \"console_url\" | query }}{{ if ne (len (label \"url\" (first $console_url ) ) ) 0}} For more information refer to {{ label \"url\" (first $console_url ) }}/settings/cluster/.{{ end }}{{ end }}" }}
+        summary: One or more cluster operators have been blocking minor version cluster upgrades for at least an hour.
+        description: In most cases, you will still be able to apply patch releases. Reason {{ "{{ with $cluster_operator_conditions := \"cluster_operator_conditions\" | query}}{{range $value := .}}{{if and (eq (label \"name\" $value) \"version\") (eq (label \"condition\" $value) \"Upgradeable\") (eq (label \"endpoint\" $value) \"metrics\") (eq (value $value) 0.0) (ne (len (label \"reason\" $value)) 0) }}{{label \"reason\" $value}}.{{end}}{{end}}{{end}}"}} For more information refer to 'oc adm upgrade'{{ "{{ with $console_url := \"console_url\" | query }}{{ if ne (len (label \"url\" (first $console_url ) ) ) 0}} or {{ label \"url\" (first $console_url ) }}/settings/cluster/{{ end }}{{ end }}" }}.
       expr: |
         max by (name, condition, endpoint) (cluster_operator_conditions{name="version", condition="Upgradeable", endpoint="metrics"} == 0)
       for: 60m
       labels:
         severity: warning
     - alert: ClusterOperatorDown
       annotations:
-        message: Cluster operator {{ "{{ $labels.name }}" }} has not been available for 10 minutes. Operator may be down or disabled, cluster will not be kept up to date and upgrades will not be possible.
+        summary: Cluster operator has not been available for 10 minutes.
```

Comment:
There is inconsistency here with 
`Cluster version operator` vs `Cluster operator` https://github.com/openshift/cluster-version-operator/pull/547/files#diff-fabad9e1d73a4f70c3d47836ed62e1982b1c6fbb947fce9a633b9cb0a98ecb24R43 

I would suggest updating all refs to use `Cluster version operator`.

Reply:
`ClusterVersionOperatorDown` is only about the cluster-version operator deployment, so `Cluster version operator` is appropriate there.

`ClusterOperatorDown` and `ClusterOperatorDegraded` are about about the conditions on ClusterVersion and ClusterOperators, so the more generic `Cluster operator` is appropriate there.